scipy/stats/_continuous_distns.py

```diff
@@ -5862,22 +5862,40 @@ def _pdf(self, x, df, nc):
         # ncx2.pdf(x, df, nc) = exp(-(nc+x)/2) * 1/2 * (x/nc)**((df-2)/4)
         #                       * I[(df-2)/2](sqrt(nc*x))
         cond = np.ones_like(x, dtype=bool) & (nc != 0)
-        return _lazywhere(cond, (x, df, nc), f=_ncx2_pdf, f2=chi2.pdf)
```

Comment:
I imagine `_ncx2_pdf` and friends are unused now, so the import should be removed. Hopefully they are not used anywhere else, too, so the code can be removed.